enterprise/api/v1/decorators.py

```diff
@@ -51,43 +51,34 @@ def wrapper(request, *args, **kwargs):
     return outer_wrapper
 
 
-def has_permission_or_group(permission, group_name, fn=None):
+def has_any_permissions(*permissions, fn=None):
     """
-    Ensure that user has permission to access the endpoint OR is part of a group that has access.
+    Decorator that allows access if the user has at least one of the specified permissions,
+    and optionally checks object-level permissions if a `fn` is provided to get the object.
     """
     def decorator(view_func):
         @wraps(view_func)
         def _wrapped_view(request, *args, **kwargs):
             user = request.user
             pk = fn(request, **kwargs) if fn else kwargs.get('pk')
-
             LOGGER.info(
                 f"[User_Permissions_Check] Checking permissions for user {user.username}, "
-                f"permission: {permission}, "
-                f"group: {group_name}, "
+                f"permission: {permissions}, "
                 f"pk: {pk}"
             )
-
             if pk:
```

Comment:
Nit: The else block is redundant here as If pk is None, and passed to has_perm, the method will handle it appropriately.

Reply:
I think this should be enough except for having if else block
`has_permission = any(user.has_perm(perm, pk) for perm in permissions)`